base/components/BG.jsx

```diff
@@ -31,7 +35,7 @@ const BG = ({image, src, children, size='cover', fullscreen, opacity}) => {
 		backgroundSize: size,
 		position: fullscreen? 'fixed' : 'absolute',
 		opacity,
-		top:0,left:0,right:0,bottom:0,
+		top:top,left:left,right:right,bottom:bottom,
```

Comment:
@dodo721 By the way, as a shorthand for e.g. `{top:top, left:left}` you can just write `{top, left}` :)